src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const resultObjectOfRules = {};
+  const arrOfRules = sourceString.split(';');
+  const arrOfFormatedRules = arrOfRules
+    .map(rule => rule
```

Comment:
you can combine it if you want
```suggestion
  const arrOfFormatedRules = sourceString
    .split(';')
    .map(rule => rule
```

Reply:
Oh, really... maybe I don't see this 😅 
Thanks for tips 🙏🏻